lib/rules/no-unused-prop-types.js

```diff
@@ -17,7 +17,7 @@ var annotations = require('../util/annotations');
 // Constants
 // ------------------------------------------------------------------------------
 
-var DIRECT_PROPS_REGEX = /^props\s*(\.|\[)/;
+var DIRECT_PROPS_REGEX = /^(props|nextProps)\s*(\.|\[)/;
```

Comment:
I did exactly like the [`prop-types` rules](https://github.com/yannickcr/eslint-plugin-react/blob/b64648521f9a7949eef4cd1a210768e906b4f3d1/lib/rules/prop-types.js#L20). It means that this one is wrong too.

What do you suggest? This regex is only used one time.